src/collections/_documentation/error-reporting/configuration/index.md

```diff
@@ -110,14 +110,14 @@ will attempt to auto-discover this value.
 {% endunsupported %}
 
 {:.config-key}
-### `allow-urls`
+### `deny-urls`
 
 {% supported browser browsernpm %}
 A list of strings or regex patterns that match error URLs which should not be sent to Sentry.  By default, all errors will be sent.
```

Comment:
```suggestion
A list of strings or regex patterns that match error URLs which should be sent to Sentry.  By default, all errors will be sent.
```

Reply:
@LSSangha Is this the mistake you pointed out earlier?

Reply:
It is. This is the fix. Thanks, Mimi.